tealer/teal/instructions/instructions.py

```diff
@@ -1228,6 +1484,14 @@ def __init__(self) -> None:
         super().__init__()
         self._version: int = 3
 
+    @property
+    def stack_pop_size(self) -> int:
+        return 3
+
+    @property
+    def stack_push_size(self) -> int:
+        return 1
+
 
 class Cover(Instruction):
```

Comment:
Add `stack_pop_size = 1 + self._idx` and `stack_push_size = 1 + self._idx` to cover instruction (?)